### DISCUSSION TITLE
pdf download not working on site

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#787] [Get Help] [answered]
I am trying again to set up the pdf output on our Italian Paintings project, and I followed the instructions on the documentation page (https://quire.getty.edu/docs-v1/multiformat-output/#pdfprint-output), but the download is not working. I ran quire build, then quire pdf, then I resized the pdf and moved it to a new content/_assets/downloads folder. There was no "downloads" folder there in my template. I changed the resource_link information in my content/_data/publication.yaml file to /downloads/my-file-name.pdf. Then I ran quire build again, and pushed my changes to GitHub. The Netlify deploy finished publishing the site without any problems, but the pdf does not download at the link in the menu. 

Does anyone have any idea why? @geealbers @Erin-Cecele Does it seem like I followed the instructions correctly? I wondered if something might be off since I didn't find a "downloads" folder in the spot where the documentation stated. Here is a link to our preview site (and you already have access to the repository):

https://italian-paintings-march2023-updates.netlify.app

I also have some questions about styling the pdf and making it a little neater all around, but I'll post those as new topics once I can get the pdf to download. 

Thank you!

**Comment by zsofiaj** [comment]
Maybe the resource_link url should instead be /_assets/downloads/my-file-name.pdf? I will try that as well.

**Reply by zsofiaj** [reply]
But that didn't work either.

**Reply by Erin-Cecele** [reply]
Thank you, @zsofiaj, for finding this issue. We have moved the conversation to our Issues Tracker. Please check issue #797 for further updates.

**Reply by zsofiaj** [reply]
Thanks so much, @Erin-Cecele !

**Comment by Erin-Cecele** [comment, marked as answer]
Issue #797 has been fixed per this comment from @geealbers https://github.com/thegetty/quire/issues/797#issuecomment-1652153965
